scripts/uninstall.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+#
+# Copyright 2023 Google Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# CRMint Uninstall Script
+#
+# Removes all resources associated with the Cloud Run version of CRMint.
+# Execute this script as: bash $HOME/crmint/scripts/uninstall.sh
+
+function execute_command() {
+    DESCRIPTION=$1
+    COMMAND=$2
+
+    echo "Attempting: $DESCRIPTION"
+    eval "$COMMAND"
+    
+    # Check the exit status
+    if [ $? -eq 0 ]; then
+        echo "$DESCRIPTION - Completed successfully."
+    else
+        echo "$DESCRIPTION - Encountered an error."
+    fi
+    echo "-----------------------------------"
+}
+
+CURRENT_REGION=$(gcloud sql instances describe crmint-3-db --format="value(region)")
+
+if [ -z "$CURRENT_REGION" ]; then
+    CURRENT_REGION="us-central1"
+fi
+
+echo "We detected your region as: $CURRENT_REGION"
+read -p "Do you want to continue with this region? (yes/no) " RESPONSE
+
+if [[ ! "$RESPONSE" =~ ^[Yy][Ee][Ss]$ ]]; then
+    echo "Please enter the region:"
+    read REGION
+else
+    REGION=$CURRENT_REGION
+fi
+
+PROJECT_ID=$(gcloud config list --format 'value(core.project)')
+echo "Your current project ID is set to: $PROJECT_ID"
+read -p "Do you want to continue with this project ID? (yes/no) " PROJECT_RESPONSE
+
+if [[ ! "$PROJECT_RESPONSE" =~ ^[Yy][Ee][Ss]$ ]]; then
```

Comment:
Same logic should be applied here to avoid errors for things like "yesd" or "ye"